platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/location/LocationModesActivity.java

```diff
@@ -123,32 +123,34 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
   public void onMapReady(@NonNull MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
 
-    int[] padding;
-    if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT) {
-      padding = new int[] {0, 750, 0, 0};
-    } else {
-      padding = new int[] {0, 250, 0, 0};
-    }
-
-    LocationComponentOptions options = LocationComponentOptions.builder(this)
-      .padding(padding)
-      .layerBelow("waterway-label")
-      .build();
+    mapboxMap.getStyle(style -> {
```

Comment:
Since the style is not restored anymore, we'll need to reset it here (previously was set only in the xml).